tasks.yaml

```diff
@@ -133,7 +133,7 @@ tasks:
     description: "Clone UDS Core for integration testing"
     actions:
       - cmd: rm -rf uds-core
-      - cmd: git clone --branch ${CORE_VERSION} https://github.com/defenseunicorns/uds-core.git
+      - cmd: git clone --branch 574-max_sessions https://github.com/defenseunicorns/uds-core.git
```

Comment:
We'll want to switch this to `main` temporarily once the core branch is merged, then back to `${CORE_VERSION}` (unless we have a good approach for keeping this on `main` with more dynamic yq, etc. to change versions in files.

Reply:
Done: https://github.com/defenseunicorns/uds-identity-config/issues/593